velox/connectors/hive/TableHandle.h

```diff
@@ -96,6 +98,10 @@ class HiveColumnHandle : public ColumnHandle {
     return columnType_ == ColumnType::kPartitionKey;
   }
 
+  const std::unordered_map<std::string, std::string>& columnParameters() const {
+    return columnParameters_;
+  }
+
   std::string toString() const;
 
   folly::dynamic serialize() const override;
```

Comment:
The serialize and creata methods should be enhanced to also serialize/deserialize the columnParameters_ in the HiveColumnHandle.

Reply:
Do we need to change serde and toString methods? The columnParameters_ will be set in Prestissimo code. Even tableParameters_ from HiveTableHandle class is implemented the same way.

Reply:
@nmahadevuni : Yes, serde and toString methods should always be updated when we update classes that appear in a Velox plan. 

toString is used in printPlanWithStats debugging utilities https://facebookincubator.github.io/velox/develop/debugging/print-plan-with-stats.html and the serde methods have been used in non-Presto use-cases.

tableParameters_ should be handled this way as well. Its a bug if it is not.